starter/script.js

```diff
@@ -1 +1,233 @@
 // put code here!
+window.alert("Welcome to the Magic Kingdom! The Happiest Place on Earth!" +
+"\nLet's take a tour of the park together!");
+
+window.alert("Our first stop is Main Street USA. The street is bustling with shops and entertainment. You can see the Dapper Dans singing barbershop quartet."
++ "\nFun fact: Each business name in the shop windows refers to a person who had some part in the building of Disney World.");
+
+var time = window.prompt("How many minutes should we stay to listen to the songs?");
```

Comment:
what is the data type of `time` here? Do we need to worry about type specificity later on?